sql/dql/what-is-sql/database-server-types.md

```diff
@@ -37,17 +37,17 @@ links:
 
 There are a number of RDBMS servers available, each with different strengths:
```

Comment:
I rewrote this one, you can leave it how it is in master now

Reply:
/